analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/ConfiguringLoggersBase.cs

```diff
@@ -20,6 +20,7 @@
 
 namespace SonarAnalyzer.Rules
 {
+    [Obsolete("This rule has been deprecated since 9.16")]
     public abstract class ConfiguringLoggersBase<TSyntaxKind> : TrackerHotspotDiagnosticAnalyzer<TSyntaxKind>
```

Comment:
I added the `Obsolete` attribute on the base class since the deprecation is for both `C#` and `VB.NET`